ui-components/src/page_components/listing/listing_sidebar/ListingUpdated.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react"
+import { t } from "../../../helpers/translator"
+import { Listing } from "@bloom-housing/backend-core/types"
+import moment from "moment"
+
+interface ListingUpdatedProps {
+  listing: Listing
```

Comment:
We're working to remove dependencies on the backend from ui-components, so we don't want to be adding any new ones. Can the content here just be passed in through the prop and can we remove `Listing` ?

Reply:
Updated to just pass the updated through as a prop vs the entire Listing.